packages/components/src/styles/prose/prose.stories.ts

```diff
@@ -9,720 +9,192 @@ const { generateTemplate } = storybookTemplate('sd-prose');
 const html = String.raw;
 
 /**
- * The `sd-prose` style is used to to add  typographic defaults to any vanilla HTML you don’t control, like HTML rendered from Markdown, or pulled from a CMS.
+ * Used to provide a set of `prose` classes to choose from in a richtext-editor.
```

Comment:
* Used as a set of `prose` classes to choose from in a richtext-editor.

Reply:
The set of style classes enables users to create visually engaging and dynamic content by incorporating various text styles, formatting options, and image embeds.

Reply:
Updated